File: Linguaggio_Alto_Livello/JavaScript/Hard/simple_contact_book.js
const contacts = {
  Luca: "333-1111111",
  Marta: "333-2222222",
};

contacts.Paolo = "333-3333333";

console.log("Rubrica:");
for (const [name, phone] of Object.entries(contacts)) {
  console.log(`${name}: ${phone}`);
}

File: Linguaggio_Alto_Livello/Python/Hard/simple_contact_book.py
contacts = {
    "Luca": "333-1111111",
    "Marta": "333-2222222",
}

new_name = "Paolo"
new_number = "333-3333333"
contacts[new_name] = new_number

print("Rubrica:")
for name, phone in contacts.items():
    print(f"{name}: {phone}")
